blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,11 @@ Unreleased Changes
 Changes
 =======
 
+- Added an optimization that allows to run predicates with
```

Comment:
```suggestion
- Added an optimization that allows to run `WHERE` clauses on top of derived tables containing table functions more efficiently in some cases.
```

Wouldn't refer to "standalone select items" - I think that doesn't make a lot of sense for end-users.

Reply:
thanks! updated